specs/address_book_list/01_main.rst

```diff
@@ -0,0 +1,140 @@
++--------+------------------+
+| issue  | title            |
++--------+------------------+
+| 81_    | Address Book List|
++--------+------------------+
+
+.. sectnum::
+
+.. _81: https://github.com/gnosis/safe/issues/81
+
+
+Address Book List
+=================
+
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Problem Definition
+-------------------------------
+
+- User can recognize names better than full addresses
+- User wants to associate addresses to names
+- User can see the address book entries and their details
+- User can search the address book entries
+- The address book is not synced (ie.: it is part of the local storage only)
+
+Inputs
+-----------
+
+- Address book entries stored locally (local database)
+- Search with name or Ethereum address
+
+Outputs
+------------
+
+- Formatted list to be displayed to the user. Each entry has:
+    - Identicon
+    - Name
+    - Ethereum address
```

Comment:
Do we allow 2 entries to have the same address?

Reply:
I would say this spec should describe the current state of the address book on Android. There we assume that each address is unique. Not sure about the name @fmrsabino